bin/zerobufCxx.py

```diff
@@ -962,10 +966,8 @@ def parse_members(self, fbsFile):
                 else:
                     elem_count = int(attrib[4])
                     member = FixedSizeArray(name, value_type, elem_count, self.name)
-                    if is_byte_type:
-                        property['type'] = 'string'
-                        property['media'] = {}
-                        property['media']['binaryEncoding'] = 'base64'
+                    if is_byte_type: # byte arrays are base64 strings
+                        property = _add_base64_string(property)
```

Comment:
So if I understand correctly the assignment and function return statement are redundant, since the function already modifies the property object passed to it